content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md

```diff
@@ -9,15 +9,46 @@ menu:
     parent: About the project
 ---
 
-## v1.9.4 []
+## v1.9.4 [TK]
 
 ### Features
-
-- Added `memUsage` metrics to `/debug/vars` endpoint.
-  Measures memory usage in bytes across all subscriptions.
+- Add more options for [restoring](/enterprise_influxdb/v1.9/tools/influxd-ctl/#restore) Enterprise databases.
+  Users can now:
+  restore data with a new retention policy into an existing database,
+  override the duration of a retention policy while restoring, and
+  specify a destination shard when restoring a specific shard. <!-- https://github.com/influxdata/plutonium/pull/3608 -->
+- Add `memUsage` metrics to `/debug/vars` endpoint
```

Comment:
Do we have the /debug/vars outputs documented? Can we have a link to that / should we document it?